custom_components/wundergroundpws/wunderground_data.py

```diff
@@ -62,6 +62,8 @@ class WUndergroundData:
         ],
         'fog': [
             "Fog",
+            "Foggy",
+            "Fog Early",
```

Comment:
`Fog Early` is not necessary here, as the `Early` gets stripped by using the `condition_modifiers` below, and it ends up looking like `Fog`, which is already coverred.